### DISCUSSION TITLE
Build stuck at receiving build, terminal log stuck at "[percy:core] Processing 2 snapshots..."

### DISCUSSION THREAD

**Discussion by ashleykolodziej** [percy/cli#477] [Support] [answered]
Hi there! This may be related to #476, but I thought I'd open my own discussion in case it's not. 

I'm using the [Percy Snapshot](https://docs.percy.io/docs/percy-snapshot) cli to generate screenshots from a list of 249 urls on one of our testing environments. When I ran this with 35 urls on a different set of sites, I didn't have any problems and everything worked great. 

For this list, I ran `npx percy snapshot snapshots.yml --verbose` and got the following results in the terminal:

[percy-logs.txt](https://github.com/percy/cli/files/6934668/percy-logs.txt)

The long and short of it being I've tried this list 3 times, and each time, it gets stuck on `[percy:core] Processing 2 snapshots...`. I haven't seen any change in the logs in the past half hour, and all three builds are showing as "Receiving build" in Percy. Is there anything you can see in here that might be preventing this from going through? It looked like all the screenshots were being generated successfully to me. Thank you!

**Comment by Robdel12** [comment, marked as answer]
Hey @ashleykolodziej! Thanks for reporting this -- we dug in and started working on a fix Friday. I was hoping to have it out by like noon today but it turns out this issue is a bit more involved. Basically, when we capture the DOM, we serialize iframes (if possible). When we take that serialized DOM and run asset discovery over it, it hangs because the of iframe CORS requests (we get one response and the rest are forever inflight). That's the gist of it, at least :p 

Anyways, this should be fixed by mid-week. Super big thanks for the logs! That was really helpful for us to crack this one open 😃 I created an issue to track too: #485

**Reply by ashleykolodziej** [reply]
No problem at all! Slightly related note... is there any way I could get a reset on my screenshot trial limit? I accidentally blew through 90% of it with this issue 😅
